internal/server/statvar_hierarchy_util.go

```diff
@@ -91,31 +96,53 @@ func getIgnoredSVGHelper(
 	}
 }
 
+func getSynonymMap() map[string][]string {
+	var synonyms map[string][]string
+	_, filename, _, _ := runtime.Caller(0)
+	bytes, err := ioutil.ReadFile(path.Join(path.Dir(filename), "resource/synonyms.json"))
+	if err == nil {
+		err = json.Unmarshal(bytes, &synonyms)
+		if err != nil {
+			synonyms = map[string][]string{}
+		}
+	}
+	return synonyms
+}
+
 // BuildStatVarSearchIndex builds the search index for the stat var hierarchy.
 func BuildStatVarSearchIndex(
 	rawSvg map[string]*pb.StatVarGroupNode,
 	blocklist bool) *resource.SearchIndex {
 	// map of token to map of sv/svg id to ranking information.
 	searchIndex := &resource.SearchIndex{
-		TokenSVMap:  map[string]map[string]struct{}{},
-		TokenSVGMap: map[string]map[string]struct{}{},
-		Ranking:     map[string]*resource.RankingInfo{},
+		RootTrieNode: &resource.TrieNode{},
```

Comment:
Can you add a timer on this function? Can use https://github.com/datacommonsorg/mixer/blob/master/internal/util/util.go#L393

Reply:
done

Reply:
done